bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieClient.java

```diff
@@ -423,26 +423,21 @@ public void writeComplete(int rc, long ledger, long entry, BookieSocketAddress a
         Counter counter = new Counter();
         byte hello[] = "hello".getBytes(UTF_8);
         long ledger = Long.parseLong(args[2]);
-        ThreadFactoryBuilder tfb = new ThreadFactoryBuilder();
-        ClientSocketChannelFactory channelFactory = new NioClientSocketChannelFactory(
-                Executors.newCachedThreadPool(tfb.setNameFormat(
-                        "BookKeeper-NIOBoss-%d").build()),
-                Executors.newCachedThreadPool(tfb.setNameFormat(
-                        "BookKeeper-NIOWorker-%d").build()));
+        EventLoopGroup eventLoopGroup = new NioEventLoopGroup(1);
```

Comment:
This was just on the `main()` method that is not really used in practice anywhere